investd/sources/bossa.py

```diff
@@ -0,0 +1,39 @@
+import csv
+from datetime import datetime
+from pathlib import Path
+from typing import Iterable
+
+from ..model import Action, AssetType, Currency, Transaction
+from .base import SourceBase
+
+COL_AMOUNT = "wartość"
+COL_CURRENCY = "waluta"
+COL_DATE = "data"
+COL_PAPER = "papier"
+COL_PRICE = "cena"
+COL_QUANTITY = "ilość"
```

Comment:
It's weird that Github doesn't automatically detect UTF8 encoding. :(

Reply:
it may not be utf8 actually, since the file I parse is in windows-1250